workflows/service_common.go

```diff
@@ -259,6 +261,19 @@ func (workflow *serviceWorkflow) serviceRegistryAuthenticator(authenticator comm
 		authParts := strings.Split(string(data), ":")
 
 		workflow.registryAuth = base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf("{\"username\":\"%s\", \"password\":\"%s\"}", authParts[0], authParts[1])))
+
+		// ImageBuild pull auth
+		var authConfigs2 map[string]types.AuthConfig
+		authConfigs2 = make(map[string]types.AuthConfig)
+
+		authConfigs2[strings.Split(workflow.serviceImage, ":")[0]] = types.AuthConfig{
```

Comment:
You're splitting the string twice. Assign to a variable to make it easier to read and avoid splitting twice